Fix #8377: Change number of autosaves that are stored (#8440)
OpenRCT2/OpenRCT2: data/language/en-GB.txt

STR_6272    :Stations
STR_6273    :Music
STR_6274    :Can't set colour scheme...

> STR_6275    :Autosave-Amount

Reviewer: ```suggestion
STR_6275    :Autosave amount:
```

STR_6275    :{WINDOW_COLOUR_2}Station style:
STR_6276    :{RED}{STRINGID} has guests getting stuck, possibly due to invalid ride type or operating mode.
STR_6277    :{WINDOW_COLOUR_2}Station index: {BLACK}{COMMA16}
STR_6278    :Autosave amount

> STR_6279    :Amount of how many autosaves should be kept

Reviewer: ```suggestion
STR_6279    :{SMALLFONT}{BLACK}Number of autosaves that should be kept
```
